pyhf/tensor/pytorch_backend.py

```diff
@@ -44,12 +44,19 @@ def astensor(self, tensor_in):
             try:
                 v = torch.autograd.Variable(torch.Tensor(tensor_in))
             except TypeError:
-                v = torch.autograd.Variable(tensor_in)
+                try:
+                    v = torch.autograd.Variable(tensor_in)
+                # Guard against not passing in something that is not a list
+                # or a tensor and causing a TypeError or RuntimeError
+                except TypeError:
+                    v = torch.autograd.Variable(torch.Tensor([tensor_in]))
+                except RuntimeError:
+                    v = torch.autograd.Variable(torch.Tensor([tensor_in]))
         return v.type(torch.FloatTensor)
```

Comment:
sorry for the piecemeal requests, can we make this

```
except (TypeError,RuntimeError):
    v = torch.autograd.Variable(torch.Tensor([tensor_in]))
```

Reply:
No problem! I truly appreciate the fact that you do a good review and this save us time overall. :) Fixed in commit 3f64c92.